tutorial/markdown/nodejs/quickstart-nextjs-capella/quickstart-nextjs-capella.md

```diff
@@ -0,0 +1,804 @@
+---
+# frontmatter
+path: "/tutorial-quickstart-nextjs-capella"
+title: Using Next.js with Couchbase NodeJS SDK
+short_title: Next.js with Node SDK
+description:
+  - Learn to build a basic server-side rendered webapp using Next.js and Couchbase
+  - See how you can fetch data from Couchbase without exposing API routes
+  - Extend your application using Next.js API routes to handle Create, Update, and Delete operations
+content_type: tutorial
+filter: sdk
+technology:
+  - kv
+  - index
+  - query
+tags:
+  - Next.js
+sdk_language:
+  - nodejs
+length: 45 Mins
+---
+
+<!-- [abstract] -->
+
+In this tutorial, you will learn how to connect to a Couchbase Capella cluster to create, read, update, and delete documents and how to write simple parametrized SQL++ queries.
+
+## Prerequisites
+
+To run this prebuilt project, you will need:
+
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/python-sdk/current/ref/travel-app-data-model.html) bucket loaded.
```

Comment:
The travel-sample bucket should be linked to Node not Python

Reply:
changed to nodejs